index.html

```diff
@@ -87,6 +87,80 @@ <h2 id="use-context">
   </div>
 </article>
 
+<article>
+  <h2 id="short-description">
+    <a href="#short-description">
+      Short description 
+    </a>
+  </h2>
+
+  <p>A spec description should never be longer than 40 characters. If this happens you should split it using a context.</p>
+
+  <div class="bad">
+{% highlight ruby %}
+it 'has 422 status code if an unexpected params will be added' do
+{% endhighlight %}
+  </div>
+
+  <div class="good">
+{% highlight ruby %}
+context 'when not valid' do
+  it { is_expected.to respond_with 422 }
+end
+{% endhighlight %}
+  </div>
+
+  <p>
+    In the example we removed the description related to the status code, which has been replaced by the 
+    expectation <code>is_expected</code>. If you run this test typing rspec filename you will obtain a readable 
+    output.
+  </p>
+
+  <div class="output">
+{% highlight ruby %}
+when not valid
+  it should respond with 422
+{% endhighlight %}
+  </div>
+</article>
+
+<article>
+  <h2 id="single-expectation">
+    <a href="#single-expectation">
+      Single expectation 
+    </a>
+  </h2>
+
+  <p>
+    The 'one expectation' tip is more broadly expressed as 'each test should make only one assertion'. 
```

Comment:
I don't agree with that because in most of the cases, having one expectation per `it` would cause unnecessary duplication of the steps "Arrange" and "Act", requiring way more lines and effort to cover the use cases.

Reply:
and nowadays rspec is able to show the exact line that failed in a `it` with many asserts